CfgLoadouts.hpp

```diff
@@ -69,10 +71,18 @@ class CfgLoadouts {
   // US: HK416 - Multi-Terrain Pattern "Loadouts\us_hk416_mtp_apex.hpp" (apex)
   // US: HK416 - Tropical "Loadouts\us_hk416_trop_apex.hpp" (apex)
   // US: M4 - Gray/Green "Loadouts\us_m4_ucp.hpp"
+  // US: M4 - Gray/Green "Loadouts\blumsv_us_m4_ucp.hpp" (msv)
   // US: M4 - Tan "Loadouts\us_m4_ocp.hpp"
+  // US: M4 - Tan "Loadouts\blumsv_us_m4_ocp.hpp" (msv)
   // US: M16 - M81 "Loadouts\us_m16_m81.hpp"
+  // US: M16 - Marine Pattern Desert "Loadouts\usmc_m16_marpatd.hpp"
+  // US: M16 - Marine Pattern Desert "Loadouts\blumsv_usmc_m16_marpatd.hpp" (msv)
+  // US: M16 - Marine Pattern Woodland "Loadouts\usmc_m16_marpatw.hpp"
```

Comment:
Same here, if not what is it?

If it's for "normal" loadouts (I.E. TvT loadouts) I think it should be blu_

Reply:
Normal loadouts are all led by country identifiers, ex Canadian Forces are can_ russians are ru_ ect. The USMC loadouts are from the base framework.